docs/articles_en/documentation/openvino-ir-format/operation-sets/operation-specs/movement/string-tensor-pack-15.rst

```diff
@@ -0,0 +1,134 @@
+and in this case holds values ``["Intel", "OpenVINO"]``.
+
+**Inputs**
+
+* **1**: *begins*:
+
+  * **Description**: Indices of each string's begginings. **Required.**
+  * **Shape**: 1D tensor of shape ``(batch_size)``.
+  * **Type**: *T_IDX*
+
+* **2**: *ends*:
+
+  * **Description**: Indices of each string's endings. **Required.**
+  * **Shape**: 1D tensor of shape ``(batch_size)``.
+  * **Type**: *T_IDX*
+
+* **3**: *symbols*:
+
+  * **Description**: Concatenated ``input`` strings encoded in utf-8 bytes. **Required.**
+  * **Shape**: 1D tensor of shape equal to the total length of concatenated strings.
+  * **Type**: *T*
+
+**Outputs**
+
+* **1**: *output*
+
+  * **Description**: A string tensor.
+  * **Shape**: ``(batch_size)``
+  * **Type**: *T*
+
+**Types**
+
+* *T*: ``u8``.
+* *T_IDX*: ``int64``.
+
+**Examples**
+
+*Example 1: input data as string*
+
+.. code-block:: xml
+   :force:
+
+    <layer ... type="StringTensorPack" ... >
+        <input>
+            <port id="0" precision="I64">
+                <dim>2</dim>     <!-- begins = [0, 5] -->
+            </port>
+            <port id="1" precision="I64">
+                <dim>2</dim>     <!-- ends = [5, 13] -->
+            </port>
+            <port id="2" precision="u8">
+                <dim>13</dim>    <!-- symbols = "IntelOpenVINO" -->
+            </port>
+        </input>
+        <output>
+            <port id="0" precision="u8">
+                <dim>2</dim>     <!-- output = ["Intel", "OpenVINO"] -->
+            </port>
+        </output>
+    </layer>
+
+*Example 2: input with an empty string*
+
+.. code-block:: xml
+   :force:
+
+    <layer ... type="StringTensorPack" ... >
+        <input>
+            <port id="0" precision="I64">
+                <dim>2</dim>     <!-- begins = [0, 3, 3, 8, 9] -->
+            </port>
+            <port id="1" precision="I64">
+                <dim>2</dim>     <!-- ends = [3, 3, 8, 9, 13] -->
+            </port>
+            <port id="2" precision="u8">
+                <dim>13</dim>    <!-- symbols = "OMZGenAI 2024"-->
+            </port>
+        </input>
+        <output>
+            <port id="0" precision="u8">
+                <dim>5</dim>     <!-- output = ["OMZ", "", "GenAI", " ", "2024"] -->
+            </port>
+        </output>
+    </layer>
+
+*Example 3: skipped symbols*
+
+.. code-block:: xml
+   :force:
+
+    <layer ... type="StringTensorPack" ... >
+        <input>
+            <port id="0" precision="I64">
+                <dim>2</dim>     <!-- begins = [0, 8] -->
+            </port>
+            <port id="1" precision="I64">
+                <dim>2</dim>     <!-- ends = [1, 9] -->
+            </port>
+            <port id="2" precision="u8">
+                <dim>13</dim>    <!-- symbols = "123456789"-->
+            </port>
+        </input>
+        <output>
+            <port id="0" precision="u8">
```

Comment:
Output is a string tensor, the `u8` precision is only int the input `symbols` tensor.